charts/actions-runner-controller-2/templates/deployment.yaml

```diff
@@ -48,6 +48,9 @@ spec:
         {{- with .Values.imagePullSecrets }}
         - "--auto-scaler-image-pull-secrets={{ include "actions-runner-controller-2.imagePullSecretsNames" . }}"
         {{- end }}
+        {{- if .Values.logging }}
```

Comment:
Should we expose all flags supported and just apply them all? Or at least, all that arc-2 app supports? For example, disable webhook etc

Reply:
Maybe, we can discuss this but if we decide to do it, those can be added in a separate PR